src/sentry/integrations/vercel/client.py

```diff
@@ -39,8 +43,22 @@ def get_user(self):
         return self.get_cached(self.USER_URL)["user"]
 
     def get_projects(self):
-        # TODO: we will need pagination since we are limited to 20
-        return self.get(self.PROJECTS_URL)["projects"]
+        limit = 20
+        params = {"limit": limit}
+        projects = []
+        # no one should have more than 200 projects
+        for i in range(10):
+            resp = self.get(self.PROJECTS_URL, params=params)
+            projects += resp["projects"]
+            # if we have less projects than the limit, we are done
+            if resp["pagination"]["count"] < limit:
+                return projects
+            # continue pagination but increment next by 1
+            params = params.copy()
+            params["since"] = resp["pagination"]["next"] + 1
+        # log the warning if this happens so we can look into solutions
+        logger.warn("Did not finish project pagination", extra={"team_id": self.team_id})
```

Comment:
I kind of wish we had more to log here than just the `team_id` but I'm going to assume that if we ever get here it's not going to be often

Reply:
@MeredithAnya I don't think we need anything more than a `team_id`. We can get the integration from that it shouldn't matter what org this is for.